designs/gozer/src/ghost.mjs

```diff
@@ -0,0 +1,96 @@
+export const ghost = {
+  name: 'gozer.ghost',
+  measurements: ['hpsToWaistBack', 'waistToFloor', 'head'],
+  draft: ({ measurements, Point, Path, points, paths, Snippet, snippets, sa, macro, part }) => {
+    const c = 0.55191502449351
+    const eyeSize = measurements.head * 0.0416
+    const size =
+      measurements.hpsToWaistBack + measurements.waistToFloor + measurements.head / Math.PI
+
+    points.middle = new Point(0, 0)
+    points.eyeLine = points.middle.shift(270, measurements.head / Math.PI / 2)
+    points.eyeLeft = points.eyeLine.shift(180, measurements.head * 0.13)
+    points.eyeRight = points.eyeLine.shift(0, measurements.head * 0.13)
+    points.top = new Point(0, -1 * size)
```

Comment:
I am trying to see a reason why you're drafting this by hand when this will do the same and is more than twice as fast to draft:

```js
    points.middle.addCircle(size)
    points.eyeLeft.addCircle(eyeSize)
    points.eyeRight.addCircle(eyeSize)
```

Is there a specific reason I'm missing, or was `Point.addcircle()` just not on your radar?

Reply:
I was totally unaware of that method. 

Thanks!

Reply:
Still keeping the self-drawing circle for the fabric itself, so I can do the `sa` normally

Reply:
Why not:
```js
if (sa) points.eyeLeft.addCircle(eyeSize + sa, 'fabric sa')
```

Reply:
Well, I tried `if (sa) points.middle.addCircle(eyeSize + sa).attr('class','fabric sa')` and that didn't work. So I just went back to the old way.